### DISCUSSION TITLE
Request: Keep Search Results Open

### DISCUSSION THREAD

**Discussion by sfein1000** [gsantner/markor#1663] [Ideas] [answered]
Unless I am missing how to perform the feature, I'd like it if there was a way to go back to the search results after closing a file. 
1) Perform search
2) Open file
3) Edit or not, then close file
4) Back at search results to select another file

Thanks

**Comment by harshad1** [comment]
I have created a PR to add this functionality. Could you test this and let me know if it is working as expected? [Link to APK - will install alongside markor as 'mordor'](https://github.com/gsantner/markor/suites/6309503695/artifacts/226379718).

**Reply by sfein1000** [reply]
I installed Mordor.  It worked for the search results as I wanted.  But there was one other difference.  When I open a document in Markor, at the top there is a left pointing arrow to the left of the document name which closes the doc.  In Mordor, that arrow is not there.  I closed the doc by hitting the phone GoBack button (lower right on phone).

**Comment by harshad1** [comment]
Yes. This was removed recently, in order to increase available space on the action bar. All it did was trigger 'back'.


May 2, 2022 06:53:03 sfein1000 ***@***.***>:

**Reply by sfein1000** [reply]
Makes sense.  The search worked well and was what I was looking to have done.  Thanks.

**Comment by gsantner** [comment, marked as answer]
It is now implemented, thanks @harshad1 - <https://github.com/gsantner/markor/pull/1689>. Available with next update after v2.9.0
